ParseCAN/spec/bus/frame.py

```diff
@@ -187,12 +187,19 @@ def __post_init__(self):
 
         self.frame.extend(frame)
 
-        if not isinstance(self.endianness, Endianness):
-            self.endianness = Endianness(self.endianness)
+        if self.type is None:
+            self.type = Type.from_str('uint8 big')
```

Comment:
This is a moderate programmer experience change, but I am okay with it.

Reply:
Any ideas for long term change?  Perhaps the spec should change to match the code rather than the other way around